src/native-ads/AdChoicesManager.tsx

```diff
@@ -36,7 +35,11 @@ export default class AdChoicesView extends React.Component<AdChoicesProps> {
       <AdChoicesViewContext.Consumer>
         {(placementId: AdChoicesViewContextValueType) => (
           <NativeAdChoicesView
```

Comment:
Did you test this code for `AdChoicesView` which don't use absolute positioning as well?

Reply:
yeah commit `style fix` resolves this and allow to override it.